cpp/src/decisiontree/batched-levelalgo/node.cuh

```diff
@@ -67,10 +67,11 @@ struct Node {
    *
    * @note to be called only by one thread across all participating threadblocks
    */
-  DI void makeLeaf(IdxT* n_leaves, LabelT pred) volatile {
+  DI void makeLeaf(IdxT* n_leaves, LabelT pred,
+                   DataT aux = DataT(-1)) volatile {
     info.prediction = pred;
     info.colid = Leaf;
-    info.quesval = DataT(0);          // don't care for leaf nodes
+    info.quesval = aux;
```

Comment:
Is this reuse necessary? I understand `quesval` is unused in leaf node but may be better to use separate member variable for this.

Reply:
I wanted to avoid introducing a new member variable, since we want to introduce a new data structure to store the probability distribution for multi-class classifiers.